pages/conference-day.tsx

```diff
@@ -0,0 +1,233 @@
+import React from 'react'
+import { GetServerSideProps, NextPage } from 'next'
+import dynamic from 'next/dynamic'
+import Link from 'next/link'
+import { format } from 'date-fns'
+import { useConfig } from 'Context/Config'
+import { Session } from 'config/types'
+import { Main } from 'layouts/main'
+import { getCommonServerSideProps } from 'components/utils/getCommonServerSideProps'
+import { fetchSessions } from 'components/utils/useSessions'
+import { useSessionGroups } from 'components/utils/useSessionGroups'
+import { StyledList, Text } from 'components/global/text'
+import { SafeLink } from 'components/global/safeLink'
+import { roomLocations } from 'components/venueMapData'
+import Image from 'next/image'
+
+const VenueMap = dynamic(() => import('components/venueMap'), { ssr: false })
+
+interface ConferencePageProps {
+  sessions?: Session[]
+  sessionId?: string
+}
+
+const ConferenceDayPage: NextPage<ConferencePageProps> = ({ sessions }) => {
+  const { conference, dates } = useConfig()
+  if (dates.IsInProgress) {
+    // dummy
```

Comment:
something coming here?

Reply:
Hmm, no, this was a bit of a hack I put in when it seemed like the `getServerSideProps()` call wasn't working. I'm loathe to remove it and potentially break things now, unless we can work out how to avoid that. 😬